tuplex/historyserver/thserver/templates/overview.html

```diff
@@ -228,7 +228,9 @@ <h6 class="mt-3 mb-3">Overview of the <span id="num-jobs">{{num_jobs }}</span> m
 
             // update row with data-jobid attribute matching the id
             var row = $('#job-table > tbody > tr[data-jobid="' + data.jobid +'"]');
-
+          if (row.find('td').eq(table_indices["Status"]).html() === "finished") {
```

Comment:
what does this do? can we add a comment?

Reply:
It ensures that a status update is never being called on an already finished job. Should never be true if history server is implemented correctly, unless requests get delivered out of order somehow. Comment explaining this has been added.